src/api/app/services/action_build_results_service/chart_data_extractor.rb

```diff
@@ -50,7 +33,20 @@ def sources_from_action(action)
     end
 
     def package_build_results(source_package, source_project)
-      source_package.buildresult(source_project, show_all: true).results.flat_map { |_k, v| v }
+      results = source_package.buildresult(source_project, show_all: true).results
+      results.flat_map do |pkg, build_results|
```

Comment:
Is there a reason to stuff the `LocalBuildResult` `ActiveModel` into a hash again?